.travis.yml

```diff
@@ -87,6 +87,7 @@ before_script:
 
 # command to run tests
 script:
+  - make test
```

Comment:
I don't think this has to run in all 19 jobs

Reply:
Then for what all jobs should I do it ?

Reply:
just for the default one